spec/controllers/samvera/persona/user_controller_spec.rb

```diff
@@ -156,6 +156,8 @@
 
     describe 'DELETE #destroy' do
       before :each do
+        FactoryBot.create(:checkout, user_id: user.id)
+        FactoryBot.create(:checkout, user_id: user.id, return_time: DateTime.current - 1.day)
```

Comment:
How about moving these into a context block around the test you added? This way they don't have to be run for all of the other tests which don't need them.

Reply:
I tried that but I get: `ArgumentError: User #{ user } does not belong to the manager group`. Something about trying to add the checkouts after doing the `RoleMap.replace` causes an issue with the collection level `add_manager` method.

Reply:
I missed the first time that this test in on the persona controller.  How about moving it to be on the model like the test for deleting bookmarks? https://github.com/avalonmediasystem/avalon/blob/90832296809f3684405acb7b4e44d7ed0720d92e/spec/models/user_spec.rb#L133-L139